config/raw.go

```diff
@@ -23,6 +23,9 @@ type Raw struct {
 
 	// BatchPairCount is the maximum limit for rawkv each batch get/delete request.
 	BatchPairCount int
+
+	// If true, the Server will only return keys when using Scan function
+	KeyOnlyScan bool
```

Comment:
seems it makes all scan key only. maybe it's better to add an argument for Scan API, though it breaks the compatibility. /cc @disksing

Reply:
agree, I think option is better. Consider to use variable argument to avoid capatibility problem.

Reply:
you can use a config structure as variable argument, they are not conflict.